src/map.h

```diff
@@ -1299,6 +1300,8 @@ class map
          * If false, monsters are not spawned in view of player character.
          */
         void spawn_monsters( bool ignore_sight );
+        static int snowfall;
+        static int percent_settled;
```

Comment:
Why are these here as static members, and not in `w_point`? They don't seem to belong to `map`.